### PR TITLE
feat: release with delay

### PR DESCRIPTION
**Pull request by nirLumigo**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -99,6 +103,26 @@ jobs:
       - run:
           name: run tests
           command: cd ../integration-tests && export testNode=true && npm run test
+  deploy-beta:
```
Comment (nirLumigo) [inline .circleci/config.yml]: Same as `deploy` except the last line

File: scripts/bd_to_beta.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+echo "Pushing beta version to npm"
+echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > .npmrc
+BETA_VERSION="$(npm view @lumigo/tracer dist-tags.beta)"
+BETA_VERSION=${BETA_VERSION:-0.0.0}
+npm version --no-git-tag-version $BETA_VERSION
+npm version --no-git-tag-version prepatch --preid beta
+npm publish --tag beta
+rm .npmrc
+
+echo "Release with delay"
```
Comment (nirLumigo) [inline scripts/bd_to_beta.sh]: All the other parts were copied from bd_to_prod.sh

File: .circleci/config.yml (hunk 1 above)
Comment (efimk-lu) [reply .circleci/config.yml]: Both deploy to beta and regular deploy will run when merging to master, that's not what you want to happen, right?

Comment (efimk-lu) [reply .circleci/config.yml]: The `deploy` should run only on trigger

Comment (nirLumigo) [reply .circleci/config.yml]: Yes, but the flow for triggering later deploy is still not done.

**Pull request merged**